cpp/arcticdb/version/version_map_batch_methods.hpp

```diff
@@ -178,23 +184,45 @@ inline std::shared_ptr<std::unordered_map<StreamId, AtomKey>> batch_get_specific
     const std::shared_ptr<Store>& store,
     const std::shared_ptr<VersionMap>& version_map,
     const std::map<StreamId, VersionId>& sym_versions,
-    bool include_deleted = true) {
+    BatchGetVersionOption option = BatchGetVersionOption::ALL_VER_FOUND_IN_STORAGE) {
```

Comment:
Assert on what BatchGetVersionOption we support, so that we find out earlier if someone adds a new one and doesn't implement it properly